fix spelling and grammar mistakes (#7073)
netdata/netdata: README.md

You can now change the data collection frequency at will, without losing previously collected values. A major improvement to the new database engine allows you not only to store metrics at variable granularity, but also to autoscale the time axis of the charts, depending on the data collection frequencies used during the presented time.

> You can also now monitor VM performance from one or more vCenter servers with a new [VSphere collector](https://docs.netdata.cloud/collectors/go.d.plugin/modules/vsphere/). Besides, the `proc` plugin now also collects ZRAM device performance metrics and the `apps` plugin monitors process uptime for the defined process groups.

Reviewer: I think "In addition" is the correct usage here. Please revert.

Continuing our efforts to integrate with as many existing solutions as possible, you can now directly archive metrics from Netdata to MongoDB via a new backend.

> Netdata badges now support international (UTF8) characters! We also made our URL parser smarter, not only for international character support but also for other strange API queries.

Reviewer: This comma is correct—please revert.

We now have a new, ["text-only" chart type](https://github.com/netdata/netdata/issues/5578), [cpu limits for v2 cgroups](https://github.com/netdata/netdata/issues/5850), [docker swarm metrics](https://docs.netdata.cloud/collectors/go.d.plugin/modules/docker_engine/) and improved [documentation](https://docs.netdata.cloud/).

> We continued improving the [Kubernetes helmchart](https://github.com/netdata/helmchart) with liveness probes for slaves, persistence options, a fix for a `Cannot allocate memory` issue and easy configuration for the kubelet, Kube-proxy and coredns collectors. 

Reviewer 2: kube-proxy is correct. 

This is a good [reference](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/).

Reviewer: I would prefer `kube-proxy`. Thanks for pointing this one out.

Release 1.13.0 contains 14 bug fixes and 8 improvements.

> Netdata has taken the first step into the world of Kubernetes, with a beta version of a [Helm chart](https://github.com/netdata/helmchart) for deployment to a k8s cluster and [proper naming](https://github.com/netdata/netdata/pull/5576) of the group containers. We have [big plans](https://github.com/netdata/netdata/issues/5392) for Kubernetes, so stay tuned!

Reviewer 3: they are cgroups though

-   **Zero disk I/O** - while it runs, it does not load or save anything (except `error` and `access` logs).
-   **Zero configuration** - auto-detects everything, it can collect up to 10000 metrics per server out of the box.
-   **Zero maintenance** - You just run it, it does the rest.

> -   **Zero dependencies** - it is even its web server, for its static web files and its web API (though its plugins may require additional libraries, depending on the applications monitored).

Reviewer 3: `it is even its web server` isn't very clear

Reviewer: Perhaps this should read something more like: 

```suggestion
-   **Zero dependencies** - Netdata runs a custom web server for its static web files and its web API (though its plugins may require additional libraries, depending on the applications monitored).
```

## Documentation

> The Netdata documentation is at <https://docs.netdata.cloud>. But you can also find it inside the repo, so by just navigating the repo on github, you can find all the documentation.

Reviewer 2: github should be in this form -> "GitHub"
